### PR TITLE
Upgrade flask to 0.12.4 and werkzeug to 0.14.1

### PR DESCRIPTION
**Pull request by kinnou02**
Remember to update your dependencies after the merge

### CODE REVIEW

File: source/jormungandr/jormungandr/interfaces/v1/Coord.py [hunk 1]
```diff
@@ -39,7 +39,7 @@
 from jormungandr.interfaces.v1.decorators import get_serializer
 from jormungandr.interfaces.v1.serializer import api
 import jormungandr
-from flask.ext.restful.fields import Raw
+from flask_restful.fields import Raw
```
Comment (pbougue) [inline source/jormungandr/jormungandr/interfaces/v1/Coord.py:42]: minor: 2 lines could be merged

File: source/jormungandr/jormungandr/modules/v1_routing/resources/Index.py [hunk 2]
```diff
@@ -38,13 +38,13 @@
...
 from jormungandr.protobuf_to_dict import protobuf_to_dict
-from flask.ext.restful.fields import Raw
+from flask_restful.fields import Raw
 from jormungandr import bss_provider_manager
 from jormungandr.interfaces.v1.decorators import get_serializer
 from jormungandr.interfaces.v1.serializer.api import TechnicalStatusSerializer
 from jormungandr.interfaces.v1.serializer.status import CommonStatusSerializer
 from jormungandr.interfaces.v1.fields import ListLit, beta_endpoint, context_utc, instance_status, add_common_status
-from flask.ext.restful import fields
+from flask_restful import fields
```
Comment (pbougue) [inline source/jormungandr/jormungandr/modules/v1_routing/resources/Index.py:47]: minor: weird imports (maybe worth using fields.Raw)

File: source/jormungandr/requirements.txt [hunk 3]
```diff
@@ -39,4 +39,5 @@ Flask-Cache==0.13.1
 git+https://github.com/canaltp/serpy.git@ctp
 python-json-logger==0.1.7
 jmespath==0.9.3
-shortuuid
\ No newline at end of file
+shortuuid
+click==6.7
```
Comment (pbougue) [inline source/jormungandr/requirements.txt:43]: Why that add?

**Review by pbougue — APPROVED** [review]

**Review by pbougue — APPROVED** [review]

File: source/jormungandr/requirements.txt (hunk 3 above)
Comment (kinnou02) [reply source/jormungandr/requirements.txt:43]: Because it's required by the new version of flask

**Review by nonifier — APPROVED** [review]

**Pull request merged**